app/cluster-api-provider-sidero/controllers/metalmachine_controller.go

```diff
@@ -58,7 +59,7 @@ func (r *MetalMachineReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, err
 
 	err = r.Get(ctx, req.NamespacedName, metalMachine)
 	if apierrors.IsNotFound(err) {
-		return ctrl.Result{}, nil
+		return ctrl.Result{RequeueAfter: 20 * time.Second}, nil
```

Comment:
Does it worths it to move this requeue time to some constants?

Reply:
Makes perfect sense. Thanks!